src/rest-client.ts

```diff
@@ -96,6 +96,10 @@ export class RestClient {
   acceptDust(quoteId: string): GenericAPIResponse {
     return this.requestWrapper.post(`dust/quotes/${quoteId}/accept`);
   }
+  
+  getUsdValueSnapshots(limit: number = 1000): GenericAPIResponse {
+    return this.requestWrapper.get(`wallet/usd_value_snapshots?${limit}`);
+  }
```

Comment:
I don't see this in the docs, does it actually work? I don't mind adding it if it does but would like a jsdoc that it's undocumented, e.g:
```suggestion
  /**
   * Returns .......
   * Note: undocumented.
   */
  getUsdValueSnapshots(limit: number = 1000): GenericAPIResponse {
    return this.requestWrapper.get(`wallet/usd_value_snapshots?${limit}`);
  }
```

Also, the limit param, should the URL become `..../wallet/usd_value_snapshots/?1000` or should it be `..../wallet/usd_value_snapshots/?limit=1000`?

If it's the second one, you can do this instead - the connector will do the rest
```
    return this.requestWrapper.get(`wallet/usd_value_snapshots`, { limit });
```

Reply:
I see it works and is used in the web UI. No, it's not documented. I'll add a jsdoc. 

Neat. I'll update how params are passed to the connector.

Reply:
@tiagosiebler actioned